packages/web-components/@groupby/elements-autocomplete/src/autocomplete.ts

```diff
@@ -35,7 +35,7 @@ export default class Autocomplete extends Base {
   /**
    * Autocomplete request results.
    */
-  @property({ type: Array }) results: AutocompleteResultGroup<AutocompleteSearchTermItem>[] = [];
+  @dataInitializer() @property({ type: Array })results: AutocompleteResultGroup<AutocompleteSearchTermItem>[] = [];
```

Comment:
This is missing a space.
```suggestion
  @dataInitializer() @property({ type: Array }) results: AutocompleteResultGroup<AutocompleteSearchTermItem>[] = [];
```

We should consider reformatting our decorator calls in the future to have the property they decorate on a separate line from the decorators. Maybe also have each decorator on separate lines as well.

Reply:
Added the space. For the comment about line splitting, we talked and both of our initial ideas for attaching the linter was the same. Keeping it how it is for now for consistency with other properties. 

For reference, these are the two options: 
```suggestion
  @dataInitializer() @property({ type: Array })
  results: AutocompleteResultGroup<AutocompleteSearchTermItem>[] = [];
```
```suggestion
  @dataInitializer() 
  @property({ type: Array })
  results: AutocompleteResultGroup<AutocompleteSearchTermItem>[] = [];
```